extension/src/test/fixtures/expShow/base/tableData.ts

```diff
@@ -15,7 +15,6 @@ const tableDataFixture: TableData = {
   hasConfig: true,
   hasMoreCommits: { main: true },
   hasRunningWorkspaceExperiment: true,
-  hasValidDvcYaml: true,
```

Comment:
[F] We no long show `Add Stage` (disabled or not) if there is an invalid `dvc.yaml`.